roles/matrix-backup-borg/templates/config.yaml.j2

```diff
@@ -26,6 +26,16 @@ consistency:
     - archives
 
 hooks:
+{% if matrix_backup_borg_postgresql_enabled %}
+  postgresql_databases:
+  {% for database in matrix_backup_borg_postgresql_databases %}
+  - name: {{ database }}
+    hostname: {{ matrix_backup_borg_postgresql_databases_hostname }}
+    username: {{ matrix_backup_borg_postgresql_databases_username }}
+    password: {{ matrix_backup_borg_postgresql_databases_password }}
```

Comment:
`|to_json` for `username` and `password` is probably important. Some people manually define very strong passwords, which contain special characters that may break the YAML syntax here. From what I remember, we've had similar problems before.

Using `|to_json` everywhere just to be on the same side is preferable.